k8s-install/canal-kubernetes-datastore.yaml

```diff
@@ -181,3 +196,83 @@ spec:
         - name: flannel-cfg
           configMap:
             name: canal-config
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
```

Comment:
I yanked these from that manifest, but probably missing a thing or two.

I don't see anything related to Flannel in that manifest, does that service account `calico-cni-plugin` enable the proper credentials?  Still figuring out RBAC.